docs/modules/user-guide/nav.adoc

```diff
@@ -13,6 +13,7 @@
 **** xref:adding-apply-commands-to-a-devfile.adoc[]
 **** xref:adding-composite-commands-to-a-devfile.adoc[]
 *** xref:adding-components-to-a-devfile.adoc[]
+**** xref:defining-top-level-attribute.adoc[]
```

Comment:
you'll need to add the `a` to the doc's name too: `defining-a-top-level-attribute.adoc[]`